db/migrate/20240925230422_add_samples_count_to_namespaces.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# migration to add samples count to namespaces table
+class AddSamplesCountToNamespaces < ActiveRecord::Migration[7.2]
+  def change
+    add_column :namespaces, :samples_count, :integer, default: 0
+  end
+end
```

Comment:
Much like how you had a migration to reset the sample counter for projects after adding the `samples_count` column to projects, you'll want a migration to set the current sample count of groups. If I were to seed my db on main, and then go to your branch and run the migration, my group sample counts are zero.